pkg/nimble/autoadv/shell.c

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2021 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     module_desire_ble_adv
+ * @{
+ *
+ * @file
+ * @brief       DESIRE based ble advertisement module shell commands
```

Comment:
What exactly is the purpose of this module?

Reply:
Just to have a shell to switch on/off